### PR TITLE
Dashboard readme: Ollie

### LINKED ISSUE #237: Finish Dashboard README
As a developer, I want clear documentation that outlines the installation process, the structure of the pipeline, and how to run the app, so new contributors can easily get started and understand the functionality of the app.

Update the readme to include details on:
- How everything works
- Wireframe diagram

### PR DESCRIPTION
**Pull request by olliejth**
This is just a pull request for the dashboard README despite the misleading branch name.

Closes #237

### CODE REVIEW

File: dashboard/README.md [hunk 1]
```diff
@@ -71,4 +72,111 @@ To get a coverage report execute the following;
...
+
+Shows the current location of the International Space Station (ISS) relative to a world map.
+
+`'Data Sources'`
+
+Logo images and URL links of the API data sources used.
+
+### - Page 2: Astronomy
+
+<span style="color:magenta;">Celestial Body Select Box</span>:
```
Comment (CalebMathers) [inline dashboard/README.md:106]: Good use of html for styling

File: dashboard/README.md [hunk 2]
```diff
@@ -2,8 +2,10 @@
 This folder contains the scripts to run and dockerise the dashboard for the StarWatch pipeline project. The pipeline was created using `streamlit` and queries data from a postgres database using `psycopg2`. Along with this, requests are also made to the [NASA APOD API](https://api.nasa.gov/) and [ISS API](http://open-notify.org/Open-Notify-API/ISS-Location-Now/).
 
 ## Design
+#### The StarWatch Architecture Design
 ![ERD](../assets/starwatch_architecture_diagram.png)
-![Wireframe](../assets/starwatch_dashboard_wireframe.png) Shows the design for the StarWatch dashboard.
+#### The StarWatch Dashboard Design Wireframe
+![Wireframe](../assets/starwatch_dashboard_wireframe.png)
```
Comment (AlexOBarnes) [inline dashboard/README.md:8]: Much better

File: dashboard/README.md [hunk 3]
```diff
@@ -71,4 +72,111 @@ To get a coverage report execute the following;
...
+
+<span style="color:magenta;">Phone Number Input Box</span>:
+
+Input box to insert user phone number.
+
+<span style="color:magenta;">County Multi-Select Box</span>:
+
+Multi-select box for UK counties. A user can use this box to select all of the counties that they would like to receive relevant alerts for.
+
+### - Page 5: Weather
```
Comment (CalebMathers) [inline dashboard/README.md:158]: Good to go into detail of each page and the sections

File: dashboard/README.md [hunk 4]
```diff
@@ -71,4 +72,111 @@ To get a coverage report execute the following;
...
+
+### - Page 3: Stellarium
+
+`'Interactive Night Sky View'`
+
+Shows an interactive map of the night sky that can be customised by location using a build in drag-and-drop pin map. This feature also contains several different visual filters that can be toggled using the icons at the bottom of the window. Current date and time are displayed below.
+
+### - Page 4: Subscribe
+
+<span style="color:magenta;">Desired Username Input Box</span>:
```
Comment (AlexOBarnes) [inline dashboard/README.md:142]: Nice use of markdown

File: dashboard/README.md [hunk 5]
```diff
@@ -71,4 +72,111 @@ To get a coverage report execute the following;
...
+
+Input box to insert desired user username.
+
+<span style="color:magenta;">Email Address Input Box</span>:
```
Comment (AlexOBarnes) [inline dashboard/README.md:146]: Nice use of markdown

File: dashboard/README.md [hunk 6]
```diff
@@ -71,4 +72,111 @@ To get a coverage report execute the following;
...
+
+Input box to insert user email address.
+
+<span style="color:magenta;">Phone Number Input Box</span>:
```
Comment (AlexOBarnes) [inline dashboard/README.md:150]: Nice use of markdown

File: dashboard/README.md [hunk 7]
```diff
@@ -71,4 +72,111 @@ To get a coverage report execute the following;
...
+
+Input box to insert user phone number.
+
+<span style="color:magenta;">County Multi-Select Box</span>:
```
Comment (AlexOBarnes) [inline dashboard/README.md:154]: Nice use of markdown

File: dashboard/README.md (hunk 2 above)
Comment (CalebMathers) [inline dashboard/README.md:8]: Lovely image insertion

File: dashboard/README.md [hunk 8]
```diff
@@ -71,4 +72,111 @@ To get a coverage report execute the following;
...
+
+Multi-select box for UK counties. A user can use this box to select all of the counties that they would like to receive relevant alerts for.
+
+### - Page 5: Weather
+
+<span style="color:magenta;">County Select Box</span>:
```
Comment (AlexOBarnes) [inline dashboard/README.md:160]: Nice use of markdown

**Review by CalebMathers — APPROVED** [review]

File: dashboard/README.md [hunk 9]
```diff
@@ -71,4 +72,111 @@ To get a coverage report execute the following;
...
+
+Select box that allows you to filter the visualisations on the weather page by UK county.
+
+<span style="color:magenta;">Date Select Box</span>:
```
Comment (AlexOBarnes) [inline dashboard/README.md:164]: Nice use of markdown

**Review by AlexOBarnes — APPROVED** [review]

**Pull request merged**